syntax = "proto3";

package v3;

message BigBinV1
{
    int32 status = 1;
    string binData = 2;
}